src/benchmark_register.cc

```diff
@@ -461,16 +463,77 @@ int Benchmark::ArgsCnt() const {
   return static_cast<int>(args_.front().size());
 }
 
+void Benchmark::prepare() {}
+
 //=============================================================================//
 //                            FunctionBenchmark
 //=============================================================================//
 
+// The class used to hold all Benchmarks created from static function.
```

Comment:
Nice job moving these classes out of the header.

They can probably put in an unnamed namespace now, since we don't really want to leak their symbols.

Reply:
OK, I will do this.